lib/replicator.js

```diff
@@ -355,15 +355,7 @@ class Peer {
     this.protomux.cork()
 
     this.sendSync()
-
-    const p = pages(this.core)
-
-    for (let index = 0; index < p.length; index++) {
-      this.wireBitfield.send({
-        start: index * this.core.bitfield.pageSize,
-        bitfield: p[index]
-      })
-    }
```

Comment:
ah we actually still need to send the bitfields, but only the ones with a start > contigPointer